omplbindings/scenes.go

```diff
@@ -69,44 +65,49 @@ func scene2() *config {
 		r3.Vector{X: 1., Y: -200., Z: 3.},
 		&spatialmath.R4AA{Theta: 0, RX: 0., RY: 0., RZ: 1.},
 	)
-	return &config{
-		Start:      startInput,
-		Goal:       spatialmath.NewPoseFromOrientation(goalPt, startPose.Orientation()),
-		RobotFrame: model,
-		WorldState: &commonpb.WorldState{
-			Obstacles: []*commonpb.GeometriesInFrame{
-				{
-					ReferenceFrame: "world",
-					Geometries: []*commonpb.Geometry{
-						{
-							Center: spatialmath.PoseToProtobuf(testPose),
-							GeometryType: &commonpb.Geometry_Box{
-								Box: &commonpb.RectangularPrism{DimsMm: &commonpb.Vector3{
-									X: 2000,
-									Y: 2000,
-									Z: 20,
-								}},
-							},
+	worldState, err := referenceframe.WorldStateFromProtobuf(&commonpb.WorldState{
+		Obstacles: []*commonpb.GeometriesInFrame{
+			{
+				ReferenceFrame: "world",
```

Comment:
[nit] Use the referenceframe.World constant

Reply:
changed